pkg/provider/branch/provider.go

```diff
@@ -404,7 +404,10 @@ func (b *branchENIProvider) DeleteBranchUsedByPods(nodeName string, UID string)
 
 	err := trunkENI.PushBranchENIsToCoolDownQueue(UID)
 	if err != nil {
-		return ctrl.Result{}, fmt.Errorf("failed to delte branch eni used by the pod %s", UID)
```

Comment:
seems this is the only place we call PushBranchENIsToCoolDownQueue, can we make it no return value instead?
it seems better to me that not handle the error here.